tests/testvault/testcorrect.yaml

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I know YAML is a super set of JSON, and this is valid, but we should either switch the file extension to JSON, or change it to indented.

Reply:
Fixed 👍🏻